cypress/e2e/explore/privacy.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { viewports } from "../../fixture/main.config";
+import { titles } from "../../fixture/privacy.config";
+
+viewports.forEach((viewport) => {
+  context(`/privacy-policy on ${viewport}`, () => {
+    beforeEach(() => {
+      cy.visit("/privacy-policy");
+      cy.viewport(<Cypress.ViewportPreset>viewport);
+    });
+
+    it("should have Protection section visible", () => {
+      cy.checkElementVisibilityAndText(
+        "section-title-privacy-policy-protecting",
+        "PROTECTING YOUR PRIVACY"
+      );
+      cy.checkElementVisibilityAndText(
+        "section-header-explore-dex-decentralized-exchange",
+        "Privacy Policy"
+      );
+      cy.checkElementVisibilityAndText(
+        "section-desc-explore-dex-decentralized-exchange",
+        "Discover our commitment to protecting your privacy through our strict privacy policy"
+      );
+    });
+
+    it("should have all the expected Titles visible expected text", () => {
+      titles.forEach((privacyTitle) => {
+        cy.get(`h2#${privacyTitle.id}`)
```

Comment:
What is this for?

Reply:
Those are css id's added in the code for the titles, to avoid double work with adding another IDs,
decided to use them, as I assume, they are expected to be unique always.

Reply:
Nope let's use testid